### PR TITLE
Wishlist and Cart pages

### PR DESCRIPTION
**Pull request by ttahm3d**
- Created Cart and Wishlist pages
- Added logout button

Cart page
127 0 0 1_5500_cart html

Wishlist page
127 0 0 1_5500_wishlist html

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
Looks good, no change required.

File: style.css [hunk 1]
```diff
@@ -140,6 +149,113 @@ h6 {
...
+.wishlist-icon-container {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  width: 1rem;
+  aspect-ratio: 1;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (nishasen) [inline style.css:168]: use util class here

**Review by coldpigli — APPROVED** [review]
Clean and Concise. Good to merge

**Review by krishanu2911 — APPROVED** [review]
No changes required good to be merged.

**Review by nishasen — APPROVED** [review]
Code looks clean and easy to understand. Added a single comment, please look into it.

**Pull request merged**